go.mod

```
@@ -1,6 +1,6 @@
 module github.com/NBISweden/sda-cli
 
-go 1.22.3
+go 1.24
```

```suggestion
go 1.24.0
```

Patch version needs to be in place or the tooling might get hiccups.